frontend/src/lib/components/ResizableTable/TableConfig.scss

```diff
@@ -24,13 +23,65 @@
         }
 
         &.selected {
-            background-color: var(--primary-hover);
+            background-color: #eef2ff;
```

Comment:
Is this something that could go in a var? We should try to stick to standard and globally specified colors whenever possible.

Reply:
Done, although I think this is a new color that is in Figma but has not been implemented in the UI, I ask @clarkus for confirmation.

Reply:
That's the highlight blue value we use in menus and other places. It should already exist in a variable somewhere. @Twixes would probably know for sure.

Reply:
It's only used here as far as I can see

![image](https://user-images.githubusercontent.com/7335343/173618627-932c000d-b011-42c8-a36f-1eb37aa7c6d2.png)

Reply:
Indeed, `--recording-current-event` seems to be `--primary-bg-hover`, while `--recording-current-event-dark` is `--primary-alt`. Practically every color used in the app should be defined in `global.scss`, unless it _has_ to be generated programmatically.